database/initialize.go

```diff
@@ -65,7 +65,7 @@ func HandleFirstStart() error {
 		return nil
 	}
 	utils.Logger.Info("First start detected, generating default data...", nil)
-	utils.Logger.Info("Generating first admin user...", nil)
+	utils.Logger.Info("Generating first administration user...", nil)
```

Comment:
Why did you make this one more explicit white at line 47 of administration/server.go you cutted it to "admin" ?

Reply:
This file was auto edited by IDE. This should'nt be part of the PR, and if there is a debate about wording in this file, this should create an issue.

Changes on this files are removed from this PR

Reply:
Fine 👍